config/config.go

```diff
@@ -93,17 +94,42 @@ type (
 		Node    string `yaml:"nodeName"`
 	}
 
-	Config struct {
-		Log      Log      `yaml:"log"`
-		Host     Host     `yaml:"host"`
-		Monitor  Monitor  `yaml:"monitor"`
-		Rapl     Rapl     `yaml:"rapl"`
-		Exporter Exporter `yaml:"exporter"`
-		Web      Web      `yaml:"web"`
-		Debug    Debug    `yaml:"debug"`
-		Dev      Dev      `yaml:"dev"` // WARN: do not expose dev settings as flags
+	// Platform contains settings for platform power monitoring
+	Platform struct {
+		Redfish Redfish `yaml:"redfish"`
+	}
+
+	// RedfishCollection contains collection behavior settings for Redfish
+	// Future versions may add interval configuration for periodic collection
+	// RedfishCollection struct {
+	// 	Interval  time.Duration `yaml:"interval"`  // Periodic collection interval (0 = on-demand only)
+	// }
+
+	// Redfish contains settings for Redfish BMC power monitoring
+	Redfish struct {
+		Enabled     *bool         `yaml:"enabled"`
+		NodeID      string        `yaml:"nodeID"`
+		ConfigFile  string        `yaml:"configFile"`
+		Staleness   time.Duration `yaml:"staleness"`   // Max age before forcing new collection (simplified caching)
+		HTTPTimeout time.Duration `yaml:"httpTimeout"` // HTTP client timeout for BMC requests
+	}
 
-		Kube Kube `yaml:"kube"`
+	// Experimental contains experimental features (no stability guarantees)
+	Experimental struct {
+		Platform Platform `yaml:"platform"`
+	}
+
+	Config struct {
+		Log          Log          `yaml:"log"`
+		Host         Host         `yaml:"host"`
+		Monitor      Monitor      `yaml:"monitor"`
+		Rapl         Rapl         `yaml:"rapl"`
+		Exporter     Exporter     `yaml:"exporter"`
+		Web          Web          `yaml:"web"`
+		Debug        Debug        `yaml:"debug"`
+		Dev          Dev          `yaml:"dev"` // WARN: do not expose dev settings as flags
+		Experimental Experimental `yaml:"experimental"`
```

Comment:
should we have a flag to enable experimental features? 
else below config is always visible in logs even when i have no BMC available and no intention of using redfish BMC power
```
experimental:
    platform:
        redfish:
            enabled: false
            nodeID: ""
            configFile: /etc/kepler/redfish.yaml
            staleness: 30s
            httpTimeout: 5s
```